pylint.cfg

```diff
@@ -2,7 +2,7 @@
 extension-pkg-whitelist=lxml
 
 [MESSAGES CONTROL]
-disable=missing-docstring,similarities,fixme,bad-continuation
```

Comment:
This same `bad-continuation` option also shows up in `.pre-commit-config.yaml`.

Reply:
yep, thanks. This was a half-baked push, didn't think anyone would catch it :)